go/vt/vttablet/tabletmanager/vreplication/vplayer.go

```diff
@@ -559,7 +559,12 @@ func (vp *vplayer) applyEvents(ctx context.Context, relay *relayLog) error {
 						if table != "" {
 							tableLogMsg = fmt.Sprintf(" for table %s", table)
 						}
-						log.Errorf("Error applying event%s: %s", tableLogMsg, err.Error())
+						pos := getNextPosition(items, i, j+1)
+						if pos != "" {
+							gtidLogMsg = fmt.Sprintf(" while processing position %s", pos)
+						}
+						log.Errorf("Error applying event%s%s: %s", tableLogMsg, gtidLogMsg, err.Error())
+						err = vterrors.Wrapf(err, "error applying event%s%s", tableLogMsg, gtidLogMsg)
```

Comment:
I am unsure, but shouldn't there be spaces between "event" and the two "%s"?

Reply:
No, as those are optional portions — each starting with a space. If we had a hardcoded space after "event" then in some cases we'd have trailing whitespace.